### PR TITLE
GetTablets from Vtctldclient

### PR DESCRIPTION
**Pull request by notfelineit**
## Description
Addresses https://github.com/vitessio/vitess/issues/14152 by fetching tablets from Vtctld instead of a single VTGate instance. `State` (serving state) will be supplemented by results from `Vtctld.RunHealthCheck`. 

## Related Issue(s)
https://github.com/vitessio/vitess/issues/14152

## Checklist

-   [ ] "Backport to:" labels have been added if this change should be back-ported
-   [ ] Tests were added or are not required
-   [ ] Did the new or modified tests pass consistently locally and on the CI
-   [ ] Documentation was added or is not required

### CODE REVIEW

File: go/vt/vtadmin/api_test.go [hunk 1]
```diff
@@ -50,6 +50,7 @@ import (
 
 	querypb "vitess.io/vitess/go/vt/proto/query"
 	tabletmanagerdatapb "vitess.io/vitess/go/vt/proto/tabletmanagerdata"
+	"vitess.io/vitess/go/vt/proto/topodata"
```
Comment (ajm188) [inline go/vt/vtadmin/api_test.go:53]: dupe import (`topodatapb` is below this line)

File: go/vt/vtadmin/cluster/cluster.go [hunk 2]
```diff
@@ -1264,13 +1265,12 @@ func (c *Cluster) getTablets(ctx context.Context) ([]*vtadminpb.Tablet, error) {
 
 		go func(tablet *topodatapb.Tablet) {
 			defer wg.Done()
-
 			var state vtadminpb.Tablet_ServingState
 			_, err := c.Vtctld.RunHealthCheck(ctx, &vtctldatapb.RunHealthCheckRequest{
 				TabletAlias: tablet.Alias,
 			})
 			if err != nil {
```
Comment (ajm188) [inline go/vt/vtadmin/cluster/cluster.go:1272]: just dropping a placeholder so i don't forget about the `UNKNOWN` vs `NOT_SERVING` distinction here

**Review by ajm188 — COMMENTED** [review]
looking good! just leaving a few small comments for now

**Comment by notfelineit** [comment]
Closing this in favor of https://github.com/vitessio/vitess/pull/15596/files

**Pull request closed without merging**